src/pages/index.js

```diff
@@ -217,7 +222,10 @@ const IndexPage = () => (
           </SprkStackItem>
 
           <SprkStackItem>
-            <SprkButton variant="secondary">
+            <SprkButton
+              href="https://github.com/sparkdesignsystem/spark-design-system/blob/master/CONTRIBUTING.md"
```

Comment:
dont you have to specify the 'a' for element with href?

Reply:
No, the SprkButton defaults to 'a' if the href is supplied. 

` if (href && !element) {
    TagName = 'a';
  } `